README.md

```diff
@@ -1,28 +1,28 @@
-# Website Toolbox Forums plugin for Craft CMS 3.x
+# Website Toolbox Community plugin for Craft CMS 3.x
 
-Single Sign On plugin for Website Toolbox Forums/jsConnect and CraftCMS
+Single Sign On plugin for Website Toolbox Community/jsConnect and CraftCMS
 
 ![Screenshot](./docs/img/plugin-logo.png)
 
-Related: [Website Toolboxforums for Craft 3.x](https://github.com/webtoolbox/craftcms-plugin)
+Related: [Website Toolboxcommunity for Craft 3.x](https://github.com/webtoolbox/craftcms-plugin)
 
 ## Requirements
 
 This plugin requires Craft CMS 3.0.0 or later.
 
 ## Installation
 
-To install Website Toolbox Forums, follow these steps:
+To install Website Toolbox Community, follow these steps:
 
-1. Install with Composer via `composer require websitetoolbox/websitetoolboxforum` from your project directory
-2. Install the plugin via `./craft install/plugin websitetoolboxforum` via the CLI -or- in the Craft Control Panel under Settings > Plugins
+1. Install with Composer via `composer require websitetoolbox/websitetoolboxcommunity` from your project directory
+2. Install the plugin via `./craft install/plugin websitetoolboxcommunity` via the CLI -or- in the Craft Control Panel under Settings > Plugins
 
-You can also install Website Toolbox Forums via the **Plugin Store** in the Craft CP.
+You can also install Website Toolbox Community via the **Plugin Store** in the Craft CP.
 
-Website Toolbox Forums works on Craft 3.x.
+Website Toolbox Community works on Craft 3.x.
```

Comment:
You have this type of mistake in several places

Reply:
It is done now.